### DISCUSSION TITLE
hi👋🏽

### DISCUSSION THREAD

**Discussion by SimTheGreat** [OpenMined/courses#186] [Introduce Yourself]
hi, 
I am Simi(13) just gave my project from chapter1 for submission...looking forward for the feedback

**Comment by yemikifouly** [comment]
Hello Simi! 👋 Welcome to OpenMined and the course! 🙂 I am so glad you have completed the course and submitted the project!!! How did you like the course? And how did you get interested in Private AI?

Also, it will probably take a few days for you to get the feedback on the project. So, don't panic if you don't get the feedback right away. 🙂

**Reply by SimTheGreat** [reply]
hi!
I am doing it with my mom...she showed my the intro video and I got really exited:)
I really liked the course so far and I cant wait for course2 where it is going to be more technical

**Reply by yemikifouly** [reply]
Oh wow!!! How very cool that you are taking the course with your mom! 😃

**Comment by SimTheGreat** [comment]
how long do think will it take to get the feedback? I sent my project on the 9th of jan
